docs/analysis_plan.md

```diff
@@ -0,0 +1,60 @@
+Data on wastewater concentrations of SARS-CoV-2 are available from individual treatment plants, indexed by sample collection date.
+Each wastewater treatment plant is contained within a state, and contains the metadata on the catchment area (population size) served by the wastewater treatment plant.
+We will assume all individuals served by the wastewater treatment plant reside within the state.
+We will jointly fit to all wastewater treatment plants within a state.
+
+# Models
+- `wwinference` (ww + hosp): This model will jointly fit to the daily hospital admissions at the state-level and the wastewater concentration data from the treatment plants within the state.
+- `wwinference` (hosp only): This model will be fit only to the daily hospital admissions at the state-level
+- baseline timeseries model (hosp only): This model will fit only to the daily hospital admissions at the state-level, using a time-series approach to generate forecasts
+
+# Forecasts
+Forecasts will be generated from each of the 3 models for all location-forecast dates where wastewater data is available.
+Forecasts will be generated at a daily temporal resoltuion, weekly, for 28 days ahead of the final forecast dates, from October 2024 to March 2025.
+
+Visual comparisons of forecasts overlaid with evaluation data will be used to assess the forecast performance of each model.
+
+## Evaluation
+The forecast performance will be evaluated against the final hospital admissions dataset available in June of 2025.
+Posterior samples of forecasts will be log-transformed, along with the observations the models are evaluated against.
+Forecasts will be scored using proper scoring rules, with metrics including the CRPS, coverage, and bias.
+
+We will summarise and analyse scores at the level of each individual forecast problem (a specific forecast date-location combination).
+This means we will summarise across the 28 horizon days to get an absolute score for each forecast.
+
+
+We will then present the results by analysing the distributions of absolute scores for each forecast in the following analyses (to be made into figures):
+- CRPS by model summarised across locations and forecast dates, broken down by under/over prediction and dispersion, in the form of a horizontal bar chart for each of the models
+- CRPS by model summarised across forecast dates, stratified by location (heatmap x-axis = model, y-axis = location)
+- CRPS by model summarised across locations, stratified by forecast date (line plot x-axis = date, y-axis = CRPS, color = model)
+- distribution of relative CRPS scores compared to 1. baseline time series and 2. wwinference (hosp only) (histograms, colored by model)
+
+The goal of these analyses and the resulting figures will be to assess:
+1. How does incorporating wastewater impact the forecast performance of the semi-mechanistic renewal model?
+2. How does the forecast performance of this type of model compare to a baseline time-series approach?
```

Comment:
I would demphasis 2. as its just a sanity check and minimal effort is going into that model (I assume). 

I would think about adding a second aim for unpacking variation in performance

Reply:
With the 2nd aim being focus on some things like regression based analyses  using wastewater data/metadata/summary metrics as predictors and regression on relative performance/absolute performance/ difference in forecasts?

Reply:
yes maybe but that might be too much work. I am thinking of something like understanding variation in perforanmcne in which your current planned analysis fit and then you could/would in a few more SI things to look at to expand on this - like looking at number of sites.

Reply:
Removed 2 and replaced with investigation into performance variation